src/server.js

```diff
@@ -59,7 +59,7 @@ onerror = function(m,s,l,c,e) {
 };
 </script>`;
           ctx.template.head.unshift(script);
-        } else if (ctx.path === ctx.prefix + '/_errors') {
+        } else if (ctx.path === '/_errors') {
```

Comment:
Don't we want the prefix?

Reply:
I was told that we strip the prefix, but it looks like we might only strip it for the URL, and not path? 

https://github.com/fusionjs/fusion-cli/blob/master/lib/strip-prefix.js

Reply:
(Path is probably derived from the URL, but will do some verification)

Reply:
Cool, almost forget about the stripping